pkgs/top-level/release-cross.nix

```diff
@@ -195,11 +195,19 @@ in
   ben-nanonote = mapTestOnCross systems.examples.ben-nanonote linuxCommon;
 
   /* Javacript */
-  ghcjs = mapTestOnCross systems.examples.ghcjs {
-    haskell.packages.ghcjs.hello = nativePlatforms;
-    haskell.packages.native-bignum.ghcHEAD.hello = nativePlatforms;
-    haskellPackages.hello = nativePlatforms;
-  };
+  ghcjs =
+    let
+      testPkgs = {
+        hello = nativePlatforms;
+        miso = nativePlatforms;
+        reflex-dom = nativePlatforms;
+      };
+    in
+    mapTestOnCross systems.examples.ghcjs {
+      haskell.packages.ghcjs.hello = nativePlatforms;
+      haskell.packages.native-bignum.ghcHEAD = testPkgs;
+      haskellPackages = testPkgs;
```

Comment:
whats the difference between the first and the second occurrence of hello?

Reply:
I assume you meant between `haskell.packages.ghcjs.hello` and `haskellPackages.hello` ?
I don't actually know how `haskell.packages.ghcjs.hello` works, but `--dry-run` shows me `ghcjs-8.10.7` in there. We probably want to remove it soon-ish ?